src/pages/PullRequestPage/PullCoverage/routes/IndirectChangesTab/IndirectChangedFiles/hooks/useIndirectChangedFilesTable.ts

```diff
@@ -0,0 +1,165 @@
+import { SortingState } from '@tanstack/react-table'
+import { isArray } from 'lodash'
+import isNumber from 'lodash/isNumber'
+import qs, { ParsedQs } from 'qs'
+import { useMemo, useState } from 'react'
+import { useLocation, useParams } from 'react-router-dom'
+
+import { PullSchemaType, usePull } from 'services/pull'
+
+const orderingDirection = Object.freeze({
+  desc: 'DESC',
+  asc: 'ASC',
+})
+
+const orderingParameter = Object.freeze({
+  name: 'FILE_NAME',
+  change: 'CHANGE_COVERAGE',
+  patch: 'PATCH_COVERAGE',
+  head: 'HEAD_COVERAGE',
+  missesCount: 'MISSES_COUNT',
+})
+
+function getFilters({
+  sorting,
+  flags,
+  components,
+}: {
+  sorting: Array<{ id: string; desc: boolean }>
```

Comment:
nit: thoughts on typing out the potential "id" values? Or is it something that isn't really static.

Just worried about typos / breaking changes from lines 36 - 48 over time

Reply:
It's a similar pattern we follow in other places(ex: ReposTable, FilesChangedTable). By default it should be missed lines so we should always be sorting on load regardless.

Reply:
Got it, yeah I was mostly curious about the different values id can take and if it'd be worth creating an enum / const for it